.github/dependabot.yml

```diff
@@ -0,0 +1,7 @@
+version: 2
+updates:
+- package-ecosystem: pip
```

Comment:
Why pip ?

Reply:
Sorry I use the dependabot to generate this. I don't know what package manager the repository uses. Is this wrong?

Reply:
https://dependabot.com/docs/config-file/#package_manager-required
Should be these i think
go:modules
go:dep

Reply:
I don't see any package manager in this repository. Am I missing something?

Reply:
dont know this. i only see docker container as package, dont see scorecard released as a go package (maybe we should). @dlorenc - can you help to review and see what is needed here.

Reply:
Doesn’t this Python repo have dependency on pip packages?

Reply:
I didn't see the repo, got confused with scorecards, yes criticality score is all pip.